nav/index.php

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" type="text/css" href="/css/cmn.css">
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png?v=9ByOqqx0o3">
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png?v=9ByOqqx0o3">
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png?v=9ByOqqx0o3">
+    <link rel="manifest" href="/manifest.json?v=9ByOqqx0o3">
+    <link rel="mask-icon" href="/safari-pinned-tab.svg?v=9ByOqqx0o3" color="#00a300">
+    <link rel="shortcut icon" href="/favicon.ico?v=9ByOqqx0o3">
+    <meta name="theme-color" content="#000000">
+    <title>Oberlin Hub</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="topline">
+        OBERLIN HUB
+      </div>
+      <div class="controllerlineone">
+        <div class="col">
+          <a href="remote"> <img class="buttons" src="/images/remoteicon.svg"> </a>
+          <p class="below">Screen <br> Controller</p>
```

Comment:
Not sure it matters, but just wanted to be sure to mention that using `<br>` tags is discouraged, and `<div` + CSS is preferred.